doc/tutorials/content/gasd_estimation.rst

```diff
@@ -0,0 +1,154 @@
+
+Query and train descriptors are matched using a nearest neighbor search approach. After that, for each matched object instance, a coarse pose is computed using the alignment transforms obtained from the reference frames of the respective query and train partial views. Given the transforms :math:`[\mathbf{R_{q}} | \mathbf{t_{q}}]` and :math:`[\mathbf{R_{t}} | \mathbf{t_{t}}]` that align the query and train partial views, respectively, the object coarse pose :math:`[\mathbf{R_{c}} | \mathbf{t_{c}}]` is obtained by
+
+.. math::
+
+ \begin{bmatrix}
+ \mathbf{R_{c}} & \mathbf{t_{c}} \\
+ \mathbf{0} & 1
+ \end{bmatrix}=
+ {\begin{bmatrix}
+ \mathbf{R_{q}} & \mathbf{t_{q}} \\
+ \mathbf{0} & 1
+ \end{bmatrix}}^{-1}
+ \begin{bmatrix}
+ \mathbf{R_{t}} & \mathbf{t_{t}} \\
+ \mathbf{0} & 1
+ \end{bmatrix}.
+
+The coarse pose :math:`[\mathbf{R_{c}} | \mathbf{t_{c}}]` can then be refined using the Iterative Closest Point (ICP) algorithm.
+
+Estimating GASD features
+------------------------
+
+The Globally Aligned Spatial Distribution is implemented in PCL as part of the
+`pcl_features <http://docs.pointclouds.org/trunk/group__features.html>`_
+library.
+
+The default values for GASD parameters are: use color data, :math:`m_s=6` (half size of 3), :math:`l_s=1`, :math:`m_c=4` (half size of 2) and :math:`l_c=12` and no histogram interpolation (INTERP_NONE). This results in an array of 984 float values. These are stored in a **pcl::GASDSignature984** point type. If color information is not used, the suggested parameter values are: :math:`m_s=8` (half size of 4), :math:`l_s=1` and trilinear histogram interpolation (INTERP_TRILINEAR). This results in an array of 512 float values, which may be stored in a **pcl::GASDSignature512** point type. It is also possible to use quadrilinear histogram interpolation (INTERP_QUADRILINEAR).
+
+The following code snippet will estimate a GASD shape + color descriptor for an input colored point cloud.
+
+.. code-block:: cpp
+   :linenos:
+
+   #include <pcl/point_types.h>
+   #include <pcl/features/gasd.h>
+
+   {
+     pcl::PointCloud<pcl::PointXYZRGBA>::Ptr cloud (new pcl::PointCloud<pcl::PointXYZRGBA>);
+
+     ... read, pass in or create a point cloud ...
+
+     // Create the GASD estimation class, and pass the input dataset to it
+     pcl::GASDEstimation<pcl::PointXYZRGBA, pcl::GASDSignature984> gasd;
+     gasd.setInputCloud (cloud);
+     gasd.setUseColor (true);
+     gasd.setShapeHalfGridSize (3);
+     gasd.setShapeHistsSize (1);
+     gasd.setShapeHistsInterpMethod (gasd.INTERP_NONE);
+     gasd.setColorHalfGridSize (2);
+     gasd.setColorHistsSize (12);
+     gasd.setColorHistsInterpMethod (gasd.INTERP_NONE);
+
+     // Output datasets
+     pcl::PointCloud<pcl::GASDSignature984>::Ptr descriptor (new pcl::PointCloud<pcl::GASDSignature984>);
```

Comment:
I've tried to follow the same standard of the VFH, PFH, FPFH and Normal estimation tutorials. Do you think I should change this?